### DISCUSSION TITLE
short cut of show in folder

### DISCUSSION THREAD

**Discussion by styw1234** [sumatrapdfreader/sumatrapdf#2646] [Ideas]
Can the author set a short cut of show in folder with "Alt+F3"?

**Comment by GitHubRulesOK** [comment]
current English `CmdShowInFolder` shortcut is <kbd>ALT</kbd>+<kbd>F</kbd>+<kbd>F</kbd>
try set your own to <kbd>ALT</kbd>+<kbd>F3</kbd> https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts

![image](https://user-images.githubusercontent.com/12754913/171424956-6abe0cc8-0b12-4327-a237-ea933af54521.png)

**Comment by styw1234** [comment]
Thanks a lot!
I finally set the CmdShowInFolder is F4
